mmv1/products/gkeonprem/VmwareNodePool.yaml

```diff
@@ -113,7 +113,7 @@ properties:
         required: true
         description: |
           The OS image to be used for each node in a node pool.
-          Currently `cos`, `ubuntu`, `ubuntu_containerd` and `windows` are supported.
+          Currently `cos`, `cos_cgv2`, `ubuntu`, `ubuntu_cgv2`, `ubuntu_containerd` and `windows` are supported.
```

Comment:
could you provide me a link of where it talks about the inclusion of these new node image types?

Reply:
Thanks for asking. Here is the context of this change b/348187956

Reply:
sorry I don't have access to the link, is this not made public yet?

Reply:
The public API doc also needs to be updated in parallel as well. https://cloud.google.com/anthos/clusters/docs/on-prem-api/reference/rest/v1/projects.locations.vmwareClusters.vmwareNodePools#VmwareNodeConfig

But you can refer to this doc on platform side https://cloud.google.com/kubernetes-engine/distributed-cloud/vmware/docs/how-to/user-cluster-configuration-file-latest#nodepool-osimage-field

Reply:
This was helpful, thanks!!